quickjs/build.gradle.kts

```diff
@@ -105,6 +107,13 @@ android {
         cppFlags("-fstrict-aliasing", "-DCONFIG_VERSION=\\\"${quickJsVersion()}\\\"")
       }
     }
+
+    packagingOptions {
+      // We get multiple copies of some license files via JNA, which is a transitive dependency of
+      // kotlinx-coroutines-test. Don't fail the build on these duplicates.
+      pickFirst("META-INF/AL2.0")
+      pickFirst("META-INF/LGPL2.1")
```

Comment:
We generally use `exclude` since these are useless files

Reply:
Done.